repo/rps.js

```diff
@@ -0,0 +1,27 @@
+const playGame = function (userChoice) {
+    userChoice = userChoice.toUpperCase();
```

Comment:
making the user choice all upper case is causing a bug in your code making the user always lose.
if you remove this part your game will be working the way intended.